machine1/dbus.go

```diff
@@ -75,6 +77,64 @@ func (c *Conn) initConnection() error {
 	return nil
 }
 
+func (c *Conn) getPath(method string, args ...interface{}) (dbus.ObjectPath, error) {
+	result := c.object.Call(fmt.Sprintf("%s%s", dbusInterface, method), 0, args...)
```

Comment:
I feel a bit strange that the caller has to pass ".method" instead of having the dot introduced here in the sprintf. Is this following some idiomatic or pre-existing pattern?